src/wlapplication.cc

```diff
@@ -454,6 +452,59 @@ WLApplication::~WLApplication() {
 	SDL_Quit();
 }
 
+void WLApplication::update_ui_theme(const UpdateThemeAction action, std::string arg) {
```

Comment:
Not sure, whether this function should be here. It would better fit into the style_manager next to the actual template_dir, but the dependency to AddOns prevents it from being there. Maybe in logic/addons?